duo_wordpress.php

```diff
@@ -156,6 +156,11 @@ function duo_auth_enabled(){
             return false; //allows the XML-RPC protocol for remote publishing
         }
 
+        if (defined('REST_REQUEST') && REST_REQUEST && duo_get_option('duo_rest', '') == 'off') {
```

Comment:
@JohnMaguire I'm going to be honest, it's been months since I issued this PR (6 months is a lifetime in some respects), I'd suggest you or someone at Duo using this as a starting place and developing it out - I can't get into the mindset where I was then - and I no longer have the need for this at this juncture, nor do I have the time unfortunately.

It's not the answer I want to give - but I'd imagine you need a way to poke through DUO when making an API request, since it should be a fairly common occurrence especially the more robust the API becomes.